cmake/Modules/FindAzureSDK_EP.cmake

```diff
@@ -121,14 +118,13 @@ if (NOT AZURESDK_FOUND)
           -DBUILD_SHARED_LIBS=OFF
           -DBUILD_TESTS=OFF
           -DBUILD_SAMPLES=OFF
+          -DCURL_NO_CURL_CMAKE=ON
```

Comment:
per your earlier comment in source above, CURL_NO_CURL_CMAKE not available with default cmake of vs2017 (cmake version 3.12.18081601-MSVC_2)
That said, does appear to build with vs2017 with warnings, and whatever azure unit tests are enabled pass.

Reply:
Good point, I'll change the comment -- `CURL_NO_CURL_CMAKE` should simply be ignored on CMake < 3.17 (and CMake's built-in `FindCURL` will set the expected variables).